pubsublite/internal/wire/subscriber_test.go

```diff
@@ -929,6 +935,15 @@ func TestSinglePartitionSubscriberStopDuringAdminSeek(t *testing.T) {
 	}
 }
 
+func verifyPartitionsActive(t *testing.T, sub Subscriber, want bool, partitions ...int) {
+	t.Helper()
+	for _, p := range partitions {
+		if got := sub.PartitionActive(p); got != want {
```

Comment:
If the Subscriber can return the entire set of partitions, verifying expectations is simpler; i.e. you do not need to check for assigned & unassigned partitions. You can just do got == wanted.

Reply:
Ack; as above.

The previous validation essentially did as you suggested, but I changed it to call the new `PartitionActive()` function.